Scala/syntax_test_scala.scala

```diff
@@ -550,6 +551,7 @@ type Foo = Bar[A] forSome { type A }
 =>     // this is here to act as a random terminator to the above partial syntax
 
    case class Thingy(abc: Int) extends Other
+// ^^^^^^^^^^^^^^^^^ meta.class.identifier.scala
```

Comment:
It seems odd that the `case` keyword would be considered part of the `meta.class` scope.